packages/dtslint/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@definitelytyped/dtslint",
+  "version": "0.0.94-next.1",
+  "description": "Runs tests on TypeScript definition files",
+  "files": [
+    "bin",
+    "dt.json",
+    "dtslint.json",
+    "dtslint-expect-only.json"
+  ],
+  "main": "bin",
+  "bin": "./bin/index.js",
+  "contributors": [
+    "Nathan Shively-Sanders <nathansa@microsoft.com> (https://github.com/sandersn)",
+    "Andy Hanson <andy-ms@microsoft.com> (https://github.com/andy-ms)",
+    "Dan Vanderkam <danvdk@gmail.com> (https://github.com/danvk)"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Microsoft/dtslint.git"
+  },
+  "scripts": {
+    "watch": "tsc --watch",
+    "build": "tsc",
+    "lint": "eslint --ext ts src",
+    "test": "npm run build && node test/test.js",
+    "prepublishOnly": "npm run build && npm run test && npm run lint"
```

Comment:
I _think_ this is already taken care of by lerna/CI scripts

Reply:
I think it was already obsolete since @orta added push-to-new-version-to-master publishing.